src/tabs/styled-components.js

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index.js';
+import type {SharedStylePropsT} from './types.js';
+import {ORIENTATION} from './constants.js';
+
+/**
+ * Main component container element
+ */
+export const Root = styled('div', (props: SharedStylePropsT) => {
+  const {$orientation} = props;
+  return {
+    display: 'flex',
+    flexDirection: $orientation === ORIENTATION.vertical ? 'row' : 'column',
+  };
+});
+
+export const Tab = styled('div', (props: SharedStylePropsT) => {
+  const {
+    $disabled,
+    $active,
+    $orientation,
+    $theme: {colors, sizing, typography},
+  } = props;
+  let style = {
+    ...typography.font300,
+    boxSizing: 'border-box',
+    color: $active ? colors.black : colors.mono800,
+    cursor: $disabled ? 'not-allowed' : 'pointer',
+    height: sizing.scale1200,
+    paddingTop: sizing.scale550,
+    paddingLeft: sizing.scale300,
+    paddingRight: sizing.scale300,
+    marginLeft: sizing.scale200,
+    marginRight: sizing.scale200,
+    borderBottom:
+      $orientation === ORIENTATION.horizontal && $active
```

Comment:
We still need to make an active tab distinguished for the vertical orientation. How about we set the right border for the vertically positioned and active tab. 
I didn't find a design for a vertical case but maybe @nguyensomniac can provide more design details on it?

Reply:
Yep, would love some ideas/details for highlighting active vertical tabs.
FWIW, the base ui figma only seems to show text color changes (black for active and gray for inactive tabs) and no borders for vertical tabs.

Reply:
<img width="308" alt="screen shot 2018-12-17 at 11 01 00 am" src="https://user-images.githubusercontent.com/5317799/50108826-13a47c80-01eb-11e9-8020-bb4b95c8b6d8.png">

Reply:
yup, that looks right!

Reply:
Awesome. Marking this as resolved then